### PR TITLE
KRV-674 : Tenant documentation for both csi-unity and operator

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
KRV-674 : Tenant documentation for both csi-unity and operator

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by prasar6 — APPROVED** [review]

**Review by rensyct — CHANGES_REQUESTED** [review]
@karthikk92 , branch should be forked from **release-1.1** branch and not main

File: content/docs/csidriver/features/unity.md [hunk 1]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
 
 >Note: Prior to CSI Driver for unity version 2.0.0, the log level was allowed to be updated dynamically through `logLevel` attribute in the secret object.
 
+## Tenancy support for Unity NFS
+
+The CSI Unity driver version 2.1.0 and later support Tenancy feature of Unity such that user will be able to associate specific worker nodes (in the cluster) and NFS storage volumes with Tenant.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:499]: Change toi: 

The CSI Unity driver version 2.1.0 (and later versions) support the Tenancy feature of Unity such that the user will be able to associate specific worker nodes (in the cluster) and NFS storage volumes with Tenant.

File: content/docs/csidriver/features/unity.md [hunk 2]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+Prerequisites (to be manually created in Unity Array) before the driver installation:
+* Create Tenants
+* Create Pools
+* Create NAS Servers with Tenant and Pool mapping
+
+Following example describes the usage of Tenant in the NFS pod creation:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:505]: The following example describes the usage of Tenant in the NFS pod creation:

File: content/docs/csidriver/features/unity.md [hunk 3]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+
+Install the csi driver using myvalues.yaml with the TenantName as below:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:507]: Install the csi driver using myvalues.yaml with the TenantName as follows:

File: content/docs/csidriver/features/unity.md [hunk 4]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+kubeletConfigDir: /var/lib/kubelet
+controller:
+    controllerCount: 2
+    volumeNamePrefix : csivol
+snapshot:
+    snapNamePrefix: csi-snap
+tenantName: "tenant3"
+```
+
+Create storageclass with NAS-Server and the Storage-Pool associated with TenantName as below:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:521]: Create storageclass with NAS-Server and the Storage-Pool associated with TenantName as follows:

File: content/docs/csidriver/features/unity.md [hunk 5]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+    tieringPolicy: "0"
+    protocol: "NFS"
+    nasServer: "nas_5"
+    provisioner: csi-unity.dellemc.com
+    reclaimPolicy: Delete
+    volumeBindingMode: WaitForFirstConsumer
+    allowVolumeExpansion: true
+```
+
+Create the pod and pvc as below:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:545]: Create the pod and pvc as follows:

File: content/docs/csidriver/features/unity.md [hunk 6]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+              mountPath: /data0
+              name: pvcname
+      volumes: 
+        - 
+          name: pvolx0
+          persistentVolumeClaim: 
+            claimName: pvcname
+```
+
+With above usage, user will be able to create NFS pod with PVC using the NAS and the Pool associated with the added Tenants specified in SC.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:600]: With the usage shown in the example, the user will be able to create NFS pod with PVC using the NAS and the Pool associated with the added Tenants specified in SC.

File: content/docs/csidriver/features/unity.md [hunk 7]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+Note: Current feature supports **ONLY single Tenant** for all the nodes in the cluster.
+User may expect error if PVC is created from the NAS server whose pool is mapped to the different tenant not associated with this SC.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:602]: User may expect an error if PVC is created from the NAS server whose pool is mapped to the different tenant not associated with this SC.

File: content/docs/csidriver/features/unity.md [hunk 8]
```diff
@@ -494,3 +494,126 @@ kubectl edit configmap -n unity unity-config-params
...
+
+For operator based installation mention the TENANT_NAME in configmap as shown below:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/unity.md:604]: For operator based installation, mention the TENANT_NAME in configmap as shown in the following example:

**Review by rsedlock1958 — COMMENTED** [review]
Several edits have been made.

File: content/docs/csidriver/features/unity.md (hunk 8 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 7 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 2 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 3 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 4 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

File: content/docs/csidriver/features/unity.md (hunk 6 above)
Comment (karthikk92) [reply content/docs/csidriver/features/unity.md]: tc

**Pull request closed without merging**